README.md

```diff
@@ -1,9 +1,15 @@
+
+
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 ## Available Scripts
 
 In the project directory, you can run:
 
+## Node Version Requirenmt
+
+Node 10.16.0 or later version
```

Comment:
```suggestion
Node 14.0.0 or later version
```

Reply:
I checked it with 10.16.0. I chose that because create-react-app is compatible with 10.16.0 or later version.

Reply:
That's not a node version.